Makefile

```diff
@@ -8,8 +8,19 @@
 #
 # -----------------------------------------------------------------
 
+# check os for path params
+ifeq ($(OS),Windows_NT)
+	PATHSEP=;
+	FOLDERSEP=\\
+	EXTENSION=.exe
+else
+	PATHSEP=:
+	FOLDERSEP=/
+	EXTENSION=""
+endif
+
 # go env vars
-GO=$(firstword $(subst :, ,$(GOPATH)))
+GO=$(firstword $(subst $(PATHSEP), ,$(GOPATH)))
```

Comment:
GOPATH n'est plus rempli par défaut, pour se protéger tu peux faire ça :

https://github.com/rlespinasse/makefile-libs/blob/master/go.mk

Reply:
C'est Windows compatible ~/go ?

Reply:
c'est compatible `PowerShell`

Reply:
encore mieux : $HOME/go on Unix systems and %USERPROFILE%\go on Windows